portals/devportal/src/main/webapp/source/src/app/components/Shared/ApiTryOut/TryOutController.jsx

```
@@ -824,7 +838,33 @@ function TryOutController(props) {
                                         </Grid>
                                     </>
                                 )}
-
+                                {isMultipleClientSecretsAllowed && securitySchemeType === 'OAUTH' && selectedKMObject
+                                && !selectedKMObject.enableTokenHashing && (
+                                    <TextField
+                                        fullWidth
+                                        variant="outlined"
+                                        margin="normal"
+                                        type={true ? "text" : "password"}
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)